src/app/user/login/login.component.ts

```diff
@@ -7,10 +7,22 @@ import { Component, OnInit } from '@angular/core';
 })
 
 export class LoginComponent implements OnInit {
-  hide = true
+  hide = true;
+  loginError = false;
+
+  testEmail = ["rtoche@uark.edu"]
```

Comment:
makes no real difference, but when hardcoding, you can put both of these values into a method like 

```
getUserEmail() {
  ...
  return testEmail
} 
```

or a single method to return both the values if you want to return it as an object:
 `return {email: testEmail, password: testPassword}`

Reply:
Yea I agree, I have it like that right now for testing purposes. Actually, if you see code that is trivial code like this one in the future just ignore it, I tend to keep things simple and kinda messy when I'm testing stuff. The validation will become a function or maybe even a class once I start working on that.